ElvaRivle/Feature/GeQERfQ9

ElvaRivle opened the pull request:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
 MAIN
 * Fixed modal look
 * Added additional fields, although not fully functional  
--------------------------------------------------------------------
 OTHER
 * Fixed \<Select\> values for Department and Tech Stack  
 * Fixed currency input width
--------------------------------------------------------------------
 TODO
 * make file upload functional
 * implement currency logic (convert all to BAM?)
 * connect with backend

ElvaRivle committed:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
 MAIN
 * fixed error in reducer.js, comma missing
 * added some colors and mixins
--------------------------------------------------------------------
 OTHER
 * /
--------------------------------------------------------------------

ElvaRivle committed:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
 MAIN
 * Fixed modal look
 * Added additional fields, although not fully functional
--------------------------------------------------------------------
 OTHER
 * Fixed <Select> values for Department and Tech Stack
 --------------------------------------------------------------------
 TODO
 * make file upload functional
 * implement currency logic (convert all to BAM?)
 * connect with backend

ElvaRivle commented:
![image](https://github.com/becarselman/ant-colony-pharaoh-frontend/assets/46258494/b2716191-e2f6-4515-96fe-eefeef8dca66)

ElvaRivle committed:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
 MAIN
 * border style solid for currency
--------------------------------------------------------------------
 OTHER
 * /
 --------------------------------------------------------------------
 TODO
 * make file upload functional
 * implement currency logic (convert all to BAM?)
 * connect with backend

ElvaRivle committed:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
MAIN
* fixed currency input width
--------------------------------------------------------------------
OTHER
* /
--------------------------------------------------------------------
TODO
* make file upload functional
* implement currency logic (convert all to BAM?)
* connect with backend

ElvaRivle committed:
Merge branch 'develop' into ElvaRivle/Feature/GeQERfQ9

becarselman commented on src/components/_addEmployeeModal/utils/FormField.js:
```diff
@@ -13,7 +15,15 @@ const FormField = ({ item }) => {
             return <MultiSelect item={item} />
         case "button":
-            return <Button item={item} />
+            //Button won't be wrapped in <FormField>
```
Please remove unnecessary comments

becarselman replied:
add new screenshot

ElvaRivle replied:
new screenshot already added, just updated old comment with new picture

ElvaRivle committed:
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
MAIN
* header span instead of paragraph
--------------------------------------------------------------------
OTHER
* removed unused imports
* removed comments
--------------------------------------------------------------------
TODO
* make file upload functional
* implement currency logic (convert all to BAM?)
* connect with backend

ElvaRivle committed:
Merge branch 'develop' into ElvaRivle/Feature/GeQERfQ9
